docs/topics/gsoc-2025.md

```diff
@@ -48,22 +48,22 @@ Thank you! We look forward to reading your applications!
 ### Build Server Protocol: add Kotlin support [Hard, 350 hrs]
 
 The Kotlin team wants to expand official Kotlin support not only for Gradle and Maven build systems,
-but any other build system as well and support them natively in JetBrains IDE with minimal effort.
-On the other hand, we also want to provide basic Kotlin support in non-JetBrains IDE –
-one part of such support is to be able to get Kotlin specific information from any build system supporting Kotlin.
+but any other build system as well and support them natively in JetBrains IDEs with minimal effort.
+On the other hand, we also want to provide basic Kotlin support in non-JetBrains IDEs –
+one part of such support is being able to get Kotlin-specific information from any build system supporting Kotlin.
 
 The solution to these requirements could be a [Build Server Protocol](https://build-server-protocol.github.io) (BSP)
-that provides an abstraction layer between build system and IDE.
+which provides an abstraction layer between the build system and IDE.
 
-The goal of this project would be implementing a prototype which uses the BSP protocol to get all required for 
-IDEA information from a user project to be able to work with Kotlin code in the project. 
-To limit the scope of this prototype – user project will be using Gradle to build itself.
+The goal of this project would be implementing a prototype which uses the BSP protocol to get all the required 
+information for IntelliJ IDEA from a user project so that it will be possible to work with Kotlin code in the project. 
+To limit the scope of this prototype – user project will use Gradle to build itself automatically.
 
-**Skills required (preferred)**
+**Preferred skills**
```

Comment:
Why the change in header? It looks like we used the same structure for the last two years.